src/jit/x64/emit.dasc

```diff
@@ -1917,9 +1917,38 @@ static void load_call_arg(MVMThreadContext *tc, MVMJitCompiler *compiler, MVMJit
         | mov TMP6, qword WORK[arg.v.reg];
         | lea TMP6, STOOGE:TMP6->data;
         break;
+    case MVM_JIT_REG_DYNIDX:
+        | get_cur_op TMP5;
+        | xor TMP6, TMP6;
+        | mov TMP6w, word [TMP5 + arg.v.reg*2]
```

Comment:
Fwiw, the two lines above can, I think, become:
`| movzx TMP6, word [TMP5 + arg.v.reg*2];`
And, the nicer thing to do (I think) would probably be to use a two-word sized type:
`| movzs TMP6, word U16:TMP5[arg.v.reg*2];`

I *think* this is only necessary because you might call this nativecall object from separate positions in the bytecode (with the argument stored in a different location each time). In which case this makes sense, but it would probably be a good idea to add some documentation to explain why this can't be 'hardcoded'.